Documentation/REST API.md

```diff
@@ -20,6 +21,7 @@
 * [Document conversion job status](#document-conversion-job-status-api-reference)
 * [Convert video](#convert-video-api-reference)
 * [Video conversion job status](#video-conversion-job-status-api-reference)
+* [Add-Ons](#add-ons-api-reference)
```

Comment:
Why not add a link to each addon?

Also, maybe we should organize this ToC a bit better? E.g. Files, Groups, Add-ons and other sections.